Compute the first three digits of pi

fpvandoorn opened the pull request:
TO CONTRIBUTORS:

Make sure you have:

  * [x] reviewed and applied the coding style: [coding](https://github.com/leanprover/mathlib/blob/master/docs/style.md), [naming](https://github.com/leanprover/mathlib/blob/master/docs/naming.md)
  * [x] make sure definitions and lemmas are put in the right files
  * [x] make sure definitions and lemmas are not redundant

[Zulip Discussion](https://leanprover.zulipchat.com/#narrow/stream/113488-general/topic/happy.20pi.20day!)

For reviewers: [code review check list](https://github.com/leanprover/mathlib/blob/master/docs/code-review.md)

fpvandoorn committed:
feat(data/real/pi): Compute the first three digits of pi

kckennylau committed:
feat(data/real/pi): better certificates

fpvandoorn committed:
feat(data/real/pi): cleanup, more digits with slower compilation time

fpvandoorn committed:
feat(pi): move results to the appropriate files

fpvandoorn committed:
chore(data/real/pi): add header

fpvandoorn committed:
fix(data/real/pi): fix compilation errors

ChrisHughes24 commented on src/data/real/basic.lean:
```diff
@@ -514,25 +516,63 @@ by simpa using sqrt_mul_self (le_refl _)
+end
+
+/- note: if you want to conclude `x ≤ sqrt y`, then use `le_sqrt_of_sqr_le`.
+   if you have `y > 0`, consider using `le_sqrt'` -/
```
Shouldn't this comment say `x > 0`

fpvandoorn replied:
fixed

fpvandoorn committed:
fix(data/real/basic): typo

fpvandoorn committed:
feat(data/real/pi): improve performance

we now apply norm_num to solve inequalities between natural numbers. Thanks to Kenny Lau for this suggestion

fpvandoorn committed:
feat(data/real/pi): add comment to gist
